remote_command_execution_vulnerability.py

```diff
@@ -50,7 +67,8 @@ def get_stok(router_ip_address):
         return None
     key = re.findall(r'key: \'(.*)\',', r0.text)[0]
     nonce = "0_" + mac + "_" + str(int(time.time())) + "_" + str(random.randint(1000, 10000))
-    router_password = input("Enter router admin password: ")
+    router_password = "12345678"
+    router_password = input("Enter router admin password: '{}']: ".format(router_password)) or router_password

```

Comment:
What is this hardcoded password?

Reply:
It's not "hardcoded" - it's initialized and a user could change it. I used 12345678 for the tests and with the placeholder, I did not have to enter it all the time.